app/views/locations/check_answers.html.erb

```diff
@@ -0,0 +1,47 @@
+<% title = @location.name %>
+<% content_for :title, title %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: "Back",
+    href: case params[:route]
+          when "locations"
+            scheme_locations_path(@scheme)
+          else
+            :back
+          end,
+  ) %>
+<% end %>
+
+<%= form_for(@location, method: :patch, url: scheme_location_check_answers_path(@scheme, @location, route: params[:route])) do |f| %>
+  <%= render partial: "organisations/headings", locals: { main: "Check your answers", sub: "Add a location to #{@scheme.service_name}" } %>
+
+  <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+          <%= govuk_summary_list do |summary_list| %>
+              <% display_location_attributes(@location).each do |attr| %>
```

Comment:
There's too much logic in this view. Can we extract it back into a/the helper?

Reply:
deprioritised as agreed, will work on this last